src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7614,13 +7468,105 @@ public enum NormalizationForm
+        public System.Text.StringBuilder Insert(int index, byte value) { throw null; }
+        public System.Text.StringBuilder Insert(int index, char value) { throw null; }
+        public System.Text.StringBuilder Insert(int index, char[] value) { throw null; }
+        public System.Text.StringBuilder Insert(int index, char[] value, int startIndex, int charCount) { throw null; }
+        public System.Text.StringBuilder Insert(int index, decimal value) { throw null; }
+        public System.Text.StringBuilder Insert(int index, double value) { throw null; }
+        public System.Text.StringBuilder Insert(int index, short value) { throw null; }
+        public System.Text.StringBuilder Insert(int index, int value) { throw null; }
+        public System.Text.StringBuilder Insert(int index, long value) { throw null; }
+        public System.Text.StringBuilder Insert(int index, object value) { throw null; }
+        public System.Text.StringBuilder Insert(int index, System.ReadOnlySpan<char> value) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public System.Text.StringBuilder Insert(int index, sbyte value) { throw null; }
+        public System.Text.StringBuilder Insert(int index, float value) { throw null; }
+        public System.Text.StringBuilder Insert(int index, string value) { throw null; }
+        public System.Text.StringBuilder Insert(int index, string value, int count) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public System.Text.StringBuilder Insert(int index, ushort value) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public System.Text.StringBuilder Insert(int index, uint value) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public System.Text.StringBuilder Insert(int index, ulong value) { throw null; }
+        public System.Text.StringBuilder Remove(int startIndex, int length) { throw null; }
+        public System.Text.StringBuilder Replace(char oldChar, char newChar) { throw null; }
+        public System.Text.StringBuilder Replace(char oldChar, char newChar, int startIndex, int count) { throw null; }
+        public System.Text.StringBuilder Replace(string oldValue, string newValue) { throw null; }
+        public System.Text.StringBuilder Replace(string oldValue, string newValue, int startIndex, int count) { throw null; }
+        void System.Runtime.Serialization.ISerializable.GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+        public override string ToString() { throw null; }
+        public string ToString(int startIndex, int length) { throw null; }
+    }
 }
 namespace System.Threading
 {
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public partial struct CancellationToken
+    public readonly partial struct CancellationToken
```

Comment:
@stephentoub Was this just an oversight that you have not marked these as readonly in the contracts?

Reply:
Oversight on my part.